app/routes/context_window.py

```
@@ -0,0 +1,940 @@
+            task_id=task_id,
+            error=str(e)
+        )
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/{conversation_id}/compact/history", response_model=CompactionHistoryResponse)
+async def get_compaction_history(
+    conversation_id: str,
+    skip: int = Query(default=0, ge=0, description="Number of records to skip"),
+    limit: int = Query(default=20, ge=1, le=100, description="Maximum records to return"),
+    user_id: str = Depends(get_current_user)
```

Ruff (0.14.14)

[warning] Unused function argument: `user_id`

(ARG001)